.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "es2020": true
+  }
+}
```

Comment:
It looks like this has the effect of overriding instead of extending the existing config in `package.json`.  I've opened #327 to move this to `package.json` and conform with the existing rules.